manifests/manifests.go

```diff
@@ -2,68 +2,73 @@ package manifests
 
 import (
 	"bytes"
+	"embed"
 	_ "embed"
 	"fmt"
+	"io/fs"
+	"path/filepath"
 	"text/template"
+)
 
-	"github.com/pkg/errors"
+const (
+	wegoManifestsDir = "wego-app"
 )
 
-var Manifests [][]byte
+var (
+	Manifests [][]byte
+	//go:embed crds/wego.weave.works_apps.yaml
+	AppCRD []byte
+	//go:embed wego-app/*
+	wegoAppTemplates embed.FS
+)
 
-//go:embed crds/wego.weave.works_apps.yaml
-var AppCRD []byte
+type WegoAppParams struct {
+	Version   string
+	Namespace string
+}
 
-//go:embed wego-app/deployment.yaml
-var WegoAppDeployment []byte
+// GenerateWegoManifests generates wego-app manifest from a template
+func GenerateWegoAppManifests(params WegoAppParams) ([][]byte, error) {
+	manifests := [][]byte{}
 
-type deploymentParameters struct {
-	Version string
-}
+	templates, _ := fs.ReadDir(wegoAppTemplates, wegoManifestsDir)
```

Comment:
should we handle error here?

Reply:
it's very unlikely this will ever return any error because it's reading directly from memory, but it doesn't hurt adding the  err check